src/wp-includes/blocks.php

```diff
@@ -1002,6 +1002,29 @@ function set_ignored_hooked_blocks_metadata( &$parsed_anchor_block, $relative_po
 	return '';
 }
 
+/**
+ * Returns the markup for blocks hooked to the given anchor block in a specific relative position and then
+ * adds a list of hooked block types to an anchor block's ignored hooked block types.
+ *
+ * This function is meant for internal use only.
+ *
+ * @since 6.5.1
```

Comment:
We're past 6.5.1. I think it's fine if we target 6.6.0 rather than 6.5.x, as it's not a major issue.

```suggestion
 * @since 6.6.0
```